### ISSUE TITLE
Lambda config example should contain runtime

### ISSUE THREAD

**Issue by Rub21** [developmentseed/chip-n-scale-queue-arranger#19]
I am trying to run the chip-scale, it trowing the  error of `Runtime` 

```
➜  chip-n-scale-queue-arranger git:(master) ✗ yarn deploy
yarn run v1.21.1
$ AWS_SDK_LOAD_CONFIG=true kes cf deploy --kes-folder config --kes-class config/kes.js
Already Uploaded: s3://test21/test21/lambdas/5af33326c1a6ce66052c3d3bc93b6ed96aa28029-DownloadAndPredict.zip
Template saved to config/cloudformation.yml
Uploaded: s3://test21/test21/cloudformation.yml
There was an error deploying the CF stack
[/Resources/DownloadAndPredictLambdaFunction/Type/Runtime] 'null' values are not allowed in templates
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
➜  chip-n-scale-queue-arranger git:(master) ✗
```

The `Runtime` value is empty in `cloudformation.yml`

![image](https://user-images.githubusercontent.com/1152236/74863091-4aa87d80-531b-11ea-844e-3d0af94e5412.png)

**Comment by drewbo** [comment]
@Rub21 you can add this to the [lambda config](https://github.com/developmentseed/chip-n-scale-queue-arranger/blob/master/config/config.example.yml#L16); I'll update in the example config and thanks for flagging

**Comment by Rub21** [comment]
Per chat with @Geoyi ,   we should add  `runtime: python3.7`, it works!!

**Comment by drewbo** [comment]
Closed via #15
